### DISCUSSION TITLE
恩山无线论坛 模板问题反馈评论区

### DISCUSSION THREAD

**Discussion by wjf0214** [wjf0214/qd-templates#183] [模板讨论]
恩山无线论坛 模板问题反馈评论区。
有问题可以下面反馈。

**Comment by zxsczx** [comment]
大佬好，最近发现恩山无法签到了 
Failed at 1/2 request,
Fail assert: {"re": "200", "from": "status"} from success_asserts,
Response Error : HTTP 502: Bad Gateway,
Request URL: https://www.right.com.cn/forum/forum.php

**Reply by wjf0214** [reply]
走代理试试？这个好像是网站节点连接失败了。。

**Comment by hecarli555** [comment]
最近不能签到了？恩山
